syntax = "proto3";

message SensorTemperature {
  int32 id = 1;
  float temperature = 2;
  uint64 timestamp = 3;
}